### ISSUE TITLE
[Feature] Macro to send specific message via Theatre Inserts

### ISSUE THREAD

**Issue by picollo-24** [ChasarooniZ/ready-for-the-stage#3]
Would it be possible to add macro that would allow sending message via Theatre Inserts?

That would allow to make all sorts of automation macros (for example some small messages when character is hit, you cuold just automate message to say "Ouch that hurts!" But this would require combining three functions, two of which are already covered by the macros (Add to stage, and remove from stage).

**Comment by ChasarooniZ** [comment]
So Is the goal something like the following?

1. Add Character to stage
2. Character Says a thing you pass in (IE like hello or something)
3. Character leaves stage?

**Comment by picollo-24** [comment]
Yes.
However steps 1 and 3 are already kinda covered, so having macro for second step would fill gaps.

**Comment by ChasarooniZ** [comment]
Yeah, I meant more to offer a macro that could do all 3 as well, but I will uhh look into it

**Comment by picollo-24** [comment]
I've been trying to make this macro to work:

https://gist.github.com/MajorVictory/a7b34b719bee0aac33b77a2d60f62056

It is supposed to do generally what I described here, but it wasn't working on modern versions, and the macro is three years old.

I cannot do proper programming, and only read code, and some minor adjustments, so I failed with that.
But it should be decent enough starting point if you're going to look at this.

You can alos see the demonstraton how it worked here:
https://www.reddit.com/r/FoundryVTT/comments/qhpgg0/using_theatre_inserts_some_music_and_a_couple/

**Comment by ChasarooniZ** [comment]
Sounds good thank you for detailing out the interest clearly, I should be able to do it theoretically we'll see!
